core/node/genesis/src/lib.rs

```diff
@@ -436,6 +492,22 @@ pub async fn create_genesis_l1_batch(
         .mark_l2_blocks_as_executed_in_l1_batch(L1BatchNumber(0))
         .await?;
 
+    insert_base_system_contracts_to_factory_deps(&mut transaction, base_system_contracts).await?;
+    insert_system_contracts(&mut transaction, factory_deps, storage_logs).await?;
```

Comment:
I guess the function has an incorrecrt name now, right? It seems like it inserts storage logs/factory deps, but actually has very little to do with actual system contracts

Reply:
Very good point - changed.